src/quantifiers/repeat.ts

```diff
@@ -0,0 +1,19 @@
+import type { Repeat } from '../types';
+import { wrapGroup } from '../utils';
+
+export function repeat(
+  compiledChildren: string,
+  config: Repeat['config']
+): string {
+  if ('count' in config && typeof config?.count === 'number') {
+    return `${wrapGroup(compiledChildren)}{${config?.count || ''}}`;
```

Comment:
[nit] Why `|| ''`? You should be able to access `config.count` directly inside the if.